doc/advanced_ffd.rst

```diff
@@ -33,7 +33,8 @@ Using our knowledge of the wing dimensions, it's easy to create a closely-confor
 The example script is located at ``examples/c172_wing/genFFD.py``.
 
 The rendering below shows the FFD volume and the point set for the wing.
-Note that unlike the cylinder example in the previous tutorial, there are now two dimensions of control points (in the streamwise and spanwise directions).
+Note that unlike the cylinder example in the previous tutorial, there are now two dimensions in control points are free to move (in the streamwise and spanwise directions).
```

Comment:
[...] "that" are free to move [...] ?

Reply:
It previously said "Note that unlike the cylinder example in the previous tutorial, there are now two dimensions of control points (in the streamwise and spanwise directions)."
And Marco said "I think this sentence is a bit confusing, there are always three dimensions but we were enforcing symmetry and not changing the streamwise coordinates in the previous example."
Not sure how best to say that or how much detail to go into.

Reply:
point out that spanwise direction is not constrained like in cylinder example